Expanding Space/Assets/Scripts/Meteorite Spawner/MeteoriteSpawner.cs

```diff
@@ -47,21 +50,31 @@ void Update()
     // this function spawns a meteorite
     IEnumerator SpawnMeteorite()
     {
-        // sets the scale of the next meteorite
-        meteorite.transform.localScale = new Vector3(meteoriteSpawnScale, meteoriteSpawnScale, meteoriteSpawnScale);
+        
 
         // waits for seconds
         yield return new WaitForSeconds(3);
 
+        // sets the scale of the next meteorite
+        meteorite.transform.localScale = new Vector3(meteoriteSpawnScale, meteoriteSpawnScale, meteoriteSpawnScale);
+
         // makes the meteorite
         Instantiate(meteorite, new Vector3(spawnPositionMeterorite, meteoriteSpawnPositionY, 0), Quaternion.identity);
 
-        // chack if it can spwn a meteorite
+        // chack if it can spawn a meteorite
         if (canSpawnMeteorite)
         {
             //calls this function again so another meteorite will spawn
             StartCoroutine(SpawnMeteorite());
         }
         
     }
+    IEnumerator StopSpawningMeteorite()
+    {
+        // waits for seconds
+        yield return new WaitForSeconds(25);
```

Comment:
Dit zou misschien ook nog in een public variabele kunnen zodat het makkelijk is aan te passen